Formula/kedge.rb

```diff
@@ -0,0 +1,28 @@
+
+class Kedge < Formula
+  desc "Simple, Concise & Declarative Kubernetes Applications"
+  homepage "http://kedgeproject.org"
+  url "https://github.com/kedgeproject/kedge.git",
```

Comment:
Please use a tagged release, not a git tag.

Reply:
meaning https://github.com/kedgeproject/kedge/archive/v0.3.0.tar.gz